File: ShopHeroesV2/src/main/java/apiRest/database/classe/Player.java
package apiRest.database.classe;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Player {
	
	private long id;
	private String name;
	private String password;
	private int level;
	private boolean admin;
	private int golds;
	private long xp;
	private List<Material> materials;
	private List<Worker> workers;
	private List<Job> jobs;
	
}

File: shop-heroes-v2/src/app/objects/player.ts
import {Worker} from './worker';
import {Job} from './job';
import { Material } from './material';

export class Player{

    constructor(name: string, password: string){
        this.name = name;
        this.password = password;
    }

    id: number;
    name:string;
    password:string;
    level:number;
    admin:boolean;
    golds:number;
    xp:number;
    materials:Material[];
    workers:Worker[];
    jobs:Job[];
}